modin/engines/ray/pandas_on_ray/frame/axis_partition.py

```diff
@@ -49,9 +50,22 @@ def deploy_func_between_two_axis_partitions(
                 kwargs,
             )
             + tuple(partitions),
-            num_return_vals=num_splits,
+            num_return_vals=num_splits * 3,
         )
 
+    def _wrap_partitions(self, partitions):
+        if isinstance(partitions, self.instance_type):
```

Comment:
When will this if statement be triggered?

Reply:
If there is just a single objectID. this is overridden

Reply:
I will restructure the code to be more evident with a `super` call.